archive_viewer/widgets/archive_search.py

```diff
@@ -1,8 +1,8 @@
 import logging
 from typing import (List, Optional)
```

Comment:
`Optional` is now an unused import and can be removed
```suggestion
from typing import List
```

Reply:
Oh, it looks like it may have been unused before your changes. But it should still be removed.